src/main/java/com/example/ShortnerURL/Controller/PingController.java

```diff
@@ -15,16 +17,18 @@ public class PingController {
     private JdbcTemplate jdbcTemplate;
 
     @GetMapping("/ping")
-    public ResponseEntity<Map<String, String>> ping() {
+    public ResponseEntity<Object> ping() {
         try {
             jdbcTemplate.queryForObject("SELECT current_timestamp;", String.class);
-            Map<String, String> response = new HashMap<>();
-            response.put("status", "ok");
-            return new ResponseEntity<>(response, HttpStatus.OK);
+            SuccessResponseDto successResponseDto = SuccessResponseDto.builder()
+                    .status("ok")
+                    .build();
+            return new ResponseEntity<>(successResponseDto, HttpStatus.OK);
         } catch (Exception e) {
```

Comment:
catch the specific exception. 
eg: if you get SqlException, catch that, translate that into an Application exception.

provide some context around the class of error in the error response dto.